test_twitter.py

```diff
@@ -0,0 +1,10 @@
+import pytest
```

Comment:
give this module a more informative name

Reply:
I have changed the name of this module